src/serializers/index.ts

```diff
@@ -4,7 +4,7 @@ import { err, errWithCause } from 'pino-std-serializers';
 import { createOmitPropertiesSerializer } from './omitPropertiesSerializer';
 import type { SerializerFn } from './types';
 
-export const DEFAULT_OMIT_HEADER_NAMES = [
+export const DEFAULT_OMIT_HEADER_NAMES = Object.freeze([
```

Comment:
Does this need to be exported from `/src/index.ts` to be available to package consumers?

Reply:
Should be here: https://github.com/seek-oss/logger/blob/8e647f10aee097258b59fe66eb0a6af5b07c0161/src/index.ts#L9